.github/workflows/ci-dockers.yml

```diff
@@ -72,9 +72,9 @@ jobs:
           # These are the base images for PL release docker images,
           # so include at least all of the combinations in release-dockers.yml.
           - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
-          - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.6.1"}
-          - {python_version: "3.9", pytorch_version: "1.13", cuda_version: "11.7.1"}
-          - {python_version: "3.10", pytorch_version: "2.0", cuda_version: "11.7.1"}
```

Comment:
@Borda This item should be added back. It's the image used in Fabric and PL Azure GPU CI. Alternatively you can update it to use the ones you added.